model/project_ref.go

```diff
@@ -1384,7 +1384,7 @@ func GetProjectSettings(p *ProjectRef) (*ProjectSettings, error) {
 	if projectVars == nil {
 		projectVars = &ProjectVars{}
 	}
-	projectAliases, err := FindAliasesForProject(p.Id, "")
+	projectAliases, err := FindAliasesForProject(p.Id)
```

Comment:
This should use FromDB; we use GetProjectSettings to get all settings for the project page.